public/app/features/logs/components/InfiniteScroll.tsx

```diff
@@ -195,27 +210,32 @@ function canScrollTop(
   currentRange: TimeRange,
   timeZone: TimeZone,
   sortOrder: LogsSortOrder
-) {
+): AbsoluteTimeRange | false {
```

Comment:
nit, but wouldn't `): AbsoluteTimeRange | undefined` be better here. As this function is getter of something, I would expect undefined if it is not able to create it.

Reply:
Makes sense! Done.